packages/types/executor/index.ts

```diff
@@ -0,0 +1,26 @@
+import {Store} from '../mongoose-store';
+import { Step } from '../step';
+import { Task, TaskStepEventData } from '../task';
+import { Workflow } from '../workflow';
+
+interface Executor {
+  workflow: Workflow;
+}
+class ExecutorImpl implements Executor {
+  public task: Task;
+  constructor(public workflow: Workflow, public store: Store<Task>) {
```

Comment:
setting an acessor modifier on a constructor param is a shorthand for defining the member and the assignement to `this`

so `constructor(public foo: string){}` becomes `public foo:string;constructor(foo:string){this.foo=foo;}`